actors/builtin/market/market_test.go

```diff
@@ -1342,9 +1369,12 @@ func TestCronTickTimedoutDeals(t *testing.T) {
 		rt.ExpectSend(builtin.BurntFundsActorAddr, builtin.MethodSend, nil, expectedBurn, nil, exitcode.Ok)
 		actor.cronTick(rt)
 
-		actor.assertDealDeleted(rt, dealIds[0])
-		actor.assertDealDeleted(rt, dealIds[1])
-		actor.assertDealDeleted(rt, dealIds[2])
+		// a second cron tick for the same epoch should not change anything
+		actor.cronTickNoChangeBalances(rt, client, provider)
+
+		actor.assertDealDeleted(rt, dealIds[0], &deal1)
+		actor.assertDealDeleted(rt, dealIds[1], &deal2)
+		actor.assertDealDeleted(rt, dealIds[2], &deal3)
```

Comment:
I think this test checks whether 2 cron ticks in a row successfully deletes deals.

I don't think this adequately tests whether a secondary CronTick is idempotent, though. Is there a way to check the root cid of the actor's state? It should be unchanged, right?

Reply:
@wadeAlexC I've added a test to check that the DealsByEpoch cid does not change.